app/views/admin/loans/index.html.slim

```diff
@@ -0,0 +1,74 @@
+- content_for :title do
```

Comment:
When I load this page I'm getting several missing assets. e.g. /images/icons/grid/expand.gif. There are three in total. Can you duplicate?